src/main/kotlin/defineDocumentation.kt

```diff
@@ -14,26 +12,68 @@ fun defineDocumentation(workspace: Workspace) {
   val docsRoot = File(App.EXPORT_LOCATION, "docs/")
   docsRoot.mkdirs()
 
-  val w = File(docsRoot, "apidocs.md").printWriter()
-  writeAPIs(w, workspace.model)
-  w.close()
+  val containersWithGitRepos = containersWithGit(workspace.model)
+
+  File(docsRoot, "apidocs.md").printWriter().let {
+    writeAPIs(it, containersWithGitRepos)
+    it.close()
+  }
+
+  File(docsRoot, "dependencies.md").printWriter().let {
+    writeDependencies(it, containersWithGitRepos)
+    it.close()
+  }
 
   val template = AutomaticDocumentationTemplate(workspace)
   template.addSections(docsRoot)
 }
 
-private fun writeAPIs(w: PrintWriter, model: Model) {
+private fun writeAPIs(w: PrintWriter, containersWithGit: List<Container>) {
   w.println("## Published APIs")
   w.println("")
 
   w.println("| Software System | API name | Purpose | Links |")
   w.println("| --- | --- | --- | --- |")
 
-  containersWithAPIDocs(model)
+  containersWithAPIDocs(containersWithGit)
     .also { println("[defineDocumentation] Found ${it.size} APIs with documentation URLs") }
     .forEach { writeAPI(w, it) }
 }
 
+private fun writeDependencies(w: PrintWriter, containersWithGit: List<Container>) {
```

Comment:
Might be worth looking at a Markdown generation library for these docs?

Reply:
Definitely!